automotive/odCANDataStructureGenerator/org.opendavinci.candatamodel/src/org/opendavinci/generator/CANDataModelGenerator.xtend

```diff
@@ -27,17 +27,19 @@ import org.eclipse.emf.ecore.resource.Resource
 import org.eclipse.xtext.generator.IFileSystemAccess
 import org.eclipse.xtext.generator.IGenerator
 import org.opendavinci.canDataModel.CANSignal
-import org.opendavinci.canDataModel.CANSignalMapping
+import org.opendavinci.canDataModel.CANMessage
+import org.opendavinci.canDataModel.CANMessageMapping
 import java.util.Random
-import org.opendavinci.canDataModel.Mapping
-import org.opendavinci.canDataModel.CANSignalTesting
+import org.opendavinci.canDataModel.CANSignalMapping
+import org.opendavinci.canDataModel.CANMessageTesting
 import org.opendavinci.canDataModel.ODVDFile
 
 class CANDataModelGenerator implements IGenerator {
 
 	/* This class describes all data to a defined CAN signal in a .can file. */
 	static class CANSignalDescription {
 		String m_FQDN
+        String m_name
```

Comment:
The indentation is imperfect to minimize the number of warnings (ironically, the warnings say "Inconsistent indentation") returned by xtend compilation. 
It is unclear to me why is it so, but if I follow the "cleaner" indentation that we like, each of these weird-indented lines will result in a new warning when the generator is compiled.
If this is an acceptable outcome, I will fix them.